.github/workflows/image-publish.yaml

```diff
@@ -22,6 +22,12 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Login to GitHub Container Registry
```

Comment:
Is this required?

Reply:
I removed it from the latest commit, but I dont think docker buildx has permissions to push, I was experimenting with it.

I have decided to --load it instead of push